bootcamp-milestone-2/src/app/blog/[slug]/page.tsx

```diff
@@ -0,0 +1,10 @@
+import Blog from "@/components/blog";
+import blogs from "@/app/blogData";
+
+export default function BlogPost({ params}: { params: Record<string, string>}) {
+  const slug = params.slug;
+  const blog = blogs.find(blog => blog.slug === `/blog/${slug}`);
```

Comment:
clever way of making this a dynamic route! You will improve this in milestone 3 when you actually fetch from a database